### ISSUE TITLE
Confusion over app behavior between user and root despite being in same sudo group

### ISSUE THREAD

**Issue by DancingQuanta** [Microsoft/BashOnWindows#1143]
I have been trying Git in WSL away from cygwin.
 I have executed `git init` in a directory `$HOME/repos` in my normal user `andrew` and nothing happends, below is sequences of tasks I have executed
```
10:45:24 andrew@LAPTOP-V9S562AR:~/repos
$ mkdir test
10:45:29 andrew@LAPTOP-V9S562AR:~/repos
$ cd test/
10:45:31 andrew@LAPTOP-V9S562AR:~/repos/test
$ ls
10:45:32 andrew@LAPTOP-V9S562AR:~/repos/test
$ git init
10:45:38 andrew@LAPTOP-V9S562AR:~/repos/test
$ ls -hal
total 0
drwxrwxrwx 2 andrew andrew 0 Sep 30 10:45 .
drwxrwxrwx 2 andrew andrew 0 Sep 30 10:45 ..
10:45:49 andrew@LAPTOP-V9S562AR:~/repos/test
$
```
This surprized me as doing above steps as user root successfully created a git repo despite being in same sudo group.
I should add that I have copied over my dotfiles from cygwin to home of WSL which  may have some unexpected results, even though the same dotfiles on debian my Raspberry Pi seems to be well behaved as cygwin.

Any advice guys?

**Comment by DancingQuanta** [comment]
My bad, it is to do with `hub` from Github not working for some reason. Closing

**Comment by fpqc** [comment]
How did you copy the dotfiles?  Did you copy them into %localappdata%\lxss?  That won't work.

**Comment by DancingQuanta** [comment]
Through git clone, but I have the alias git=hub in my dotfiles that
prevents git from working despite hub being installed.
I mistakenly thought that git behaved differently under root user, however
I tried my dotfiles under root directory sms that causes git to behave
badly.
So I tracked this misbehavior to this alias hub and everything works fine
without this alias.
Apparently there are some issues with hub on WSL so I left that for another
time
